devenv.nix

```diff
@@ -36,11 +34,40 @@ in {
     go = {
       enable = true;
       package = pkgs-unstable.go;
+			enableHardeningWorkaround = true;
     };
   };
   
   services.mailpit = {
     enable = true;
     package = pkgs-unstable.mailpit;
   };
+	
+	devcontainer = {
+		enable = true;
+		settings = {
+			updateContentCommand = "sudo setfacl -k /tmp && devenv test";
+			forwardPorts = [ 4173 3456 ];
+			portsAttributes = {
+				"4173" = {
+					label = "Vikunja Frontend dev server";
+				};
+				"3456" = {
+					label = "Vikunja API";
+				};
+			};
+			customizations.vscode.extensions = [
```

Comment:
Warum muss das hier nochmal wiederholt werden?

Reply:
Frag ich mich auch, aber das ist in der spec von den devcontainers so festgelegt. Glaub der Unterschied ist, dass die aus https://github.com/go-vikunja/vikunja/blob/main/.vscode/extensions.json eben nur Empfehlungen sind und das, was man hier angibt sind Extensions die während dem setup installiert werden.

Reply:
Ahh ich meinte allerdings:

https://github.com/go-vikunja/vikunja/pull/983/files/b59f9e611a597aa45ba5f535b3ae589f3e45b60d

Reply:
Yes, merkt man eigentlich gar nicht 🙂